weave

```diff
@@ -261,14 +261,14 @@ elif [ "$1" = "run" -a -z "$IS_LOCAL" ] ; then
     # we can't rely on our baked-in docker to support those arguments.
     shift 1
     if [ "$1" != "--without-dns" ] ; then
-        DNS_ARGS=$(exec_remote dns-args "$@" || true)
+        eval "$(exec_remote dns-args "$@" 2>/dev/null || true)"
```

Comment:
That's the only solution I find for getting two vars with `exec_remote dns-args`. Otherwise, I would have to do 

```
DNS_ARGS=$(exec_remote dns-args)
DNS_DOMAIN=$(exec_remote dns-domain)
```

Any idea?